src/components/buttons/HdLoaderButton.vue

```diff
@@ -76,15 +115,23 @@ export default {
       type: Number,
       default: 4,
     },
-  },
-  mounted() {
-    this.undrawAll();
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
I would also cover this guy with tests as it has an impact on how this component behaves.

Reply:
Right, but there is a separate ticket for improving its tests.
https://homeday.atlassian.net/browse/FET-56

Reply:
Yes but as you're adding things to it, I thing it is safe to cover those scenarios. The ticket mentions improving test coverage which would be "not yet covered scenarios".

Reply:
Yeah right!